extensions/wikia/MercuryApi/models/MercuryApi.class.php

```diff
@@ -116,6 +117,8 @@ public function getWikiVariables() {
 			'enableNewAuth' => $wgEnableNewAuth,
 			'homepage' => $this->getHomepageUrl(),
 			'id' => (int) $wgCityId,
+			'isCoppaWiki' => ($wgWikiDirectedAtChildrenByFounder || $wgWikiDirectedAtChildrenByStaff),
```

Comment:
This should be spacey per our PHP coding conventions, i.e. `( $wgWikiDirectedAtChildrenByFounder || $wgWikiDirectedAtChildrenByStaff )`